liquibase-core/src/main/java/liquibase/snapshot/jvm/ViewSnapshotGenerator.java

```diff
@@ -128,7 +133,11 @@ protected void addTo(DatabaseObject foundObject, DatabaseSnapshot snapshot) thro
                 for (CachedRow row : viewsMetadataRs) {
                     CatalogAndSchema catalogAndSchema = ((AbstractJdbcDatabase) database).getSchemaFromJdbcInfo(row.getString("TABLE_CAT"), row.getString("TABLE_SCHEM"));
                     View view = new View();
-                    view.setName(row.getString("TABLE_NAME"));
+                    String name = row.getString("TABLE_NAME");
+                    if (database instanceof PostgresDatabase && POSTGRES_EXCLUDED_PUBLIC_VIEWS.contains(name)) {
```

Comment:
These views can exist only in public schema, should we add into if statement check if schema is public or default one?

Reply:
good catch, pushed fix for it